app/javascript/modules/listingDetails/ListingDetailsPricingTable.tsx

```diff
@@ -206,7 +200,7 @@ const buildAccordions = (
               </span>
               <span className={"flex items-center mr-2 text-sm md:text-base"}>
                 {(() => {
-                  return occupancy?.absoluteMinIncome?.valueOf() === 0 ? (
+                  return occupancy?.absoluteMinIncome?.valueOf() <= 0 ? (
```

Comment:
Do we actually need to use `valueOf`? I know that this is existing code, but since this line is getting updated, I'm using this opportunity to discuss removing it.

Reply:
Honestly, I doubt that we need it. If you want to remove it that is good with me

Reply:
I think removing `valueOf` is a good idea, it's not normally used this way and could be confusing for other developers.